### ISSUE TITLE
Exit button in bottom drawer does not have "close" accessibility label

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1096]
From the credentials list

**Comment by cvarjao** [comment]
it works
